src/stress_scheduler.py

```diff
@@ -431,41 +433,41 @@ def model_machine(ssh_clients, container_ids_dict, experiment_inc_args, experime
         #Initialize Spark Master
         initialize_spark_experiment(experiment_args[0])
     else:
-        print 'INVALID EXPERIMENT TYPE: {}'.format(args.experiment_type)
+        logging.warning('INVALID EXPERIMENT TYPE: {}'.format(args.experiment_type))
         exit()
 
     # Notifying User CPU throttling type
     cpu_cores = args.cpu_cores
     if cpu_cores:
-        print 'Using CPU Core Throttling'
+        logging.info('Using CPU Core Throttling')
     else:
-        print 'Using CPU Quota Throttling'
+        logging.info('Using CPU Quota Throttling')
 
     # Getting increments
     if not args.increments:
         if args.resume:
-            print 'If resuming from a previous experiment, please specify increments'
+            logging.warning('If resuming from a previous experiment, please specify increments')
             exit()
         increments = [0, 20, 40, 60, 80]
     else:
         if args.only_baseline:
-            print 'Cannot specify increments when only_baseline is true'
+            logging.warning('Cannot specify increments when only_baseline is true')
             exit()
         string_increments = args.increments.split(',')
         try:
             increments = map(int, string_increments)
         except:
-            print 'ERROR: Increments must be integers'
+            logging.warning('ERROR: Increments must be integers')
             exit()
     experiment_inc_args = [increments, experiment_args]
 
     if args.resume:
-        print 'RESUME FUNCTIONALITY HAS BEEN SHELVED FOR LATER (AND IT IS ALSO OUTDATED)'
+        logging.warning('RESUME FUNCTIONALITY HAS BEEN SHELVED FOR LATER (AND IT IS ALSO OUTDATED)')
```

Comment:
This should be at the error level, it causes the program to exit

Side note, if resume is deprecated and not implemented, shouldn't it be deleted